sdk/python/lib/pulumi/x/automation/stack.py

```diff
@@ -465,18 +465,23 @@ def outputs(self) -> OutputMap:
         return outputs
 
     def history(self,
-        limit: Optional[int] = None) -> List[UpdateSummary]:
+        pageSize: Optional[int] = None,
```

Comment:
Python convention is snake_case
```suggestion
        page_size: Optional[int] = None,
```

Reply:
ah yes thanks. did a find + replace on these, updated.